### ISSUE TITLE
Things to try on CAL

### ISSUE THREAD

**Issue by guneytombak** [Sebi1128/DeepLearningProject#10]
Change number of neighbors (50 or 3)
Change distance metric (l2 or symmetric kl)
Use the index instead of probability (~KNN)
Inverse the counting (might work or might worsen)

**Comment by AronDiSc** [comment]
- [ ] resolve initialization issue @guneytombak 
- [ ] different population size (e.g. 1%, 2%, 3%) @guneytombak 
- [ ] l2 distance @guneytombak 
- [ ] Pretrained VAE for MNIST, fashion MNIST (CIFAR100) @AronDiSc 
- [ ] Dataloader MNIST, fashion MNIST @remo48 
- [x] Low dimensional representation -> plot / visualization https://scikit-learn.org/stable/modules/generated/sklearn.manifold.TSNE.html @Sebi1128 
=> Until Friday 31/12/21
